syntax = "proto3";

service Showtime {
    rpc GetScheduleByDate(Date) returns (ScheduleData) {}
    rpc GetAllSchedules(EmptyDate) returns (stream ScheduleData) {}
}

message ScheduleData {
    string date = 1;
    repeated string movies = 2;
}

message Date {
    string date = 1;
}

message EmptyDate {}
